pkgs/development/tools/misc/gede/default.nix

```
@@ -1,12 +1,12 @@
-{ stdenv, fetchurl, makeWrapper, python, qmake, ctags, gdb }:
+{ mkDerivation, lib, fetchurl, makeWrapper, python, qmake, ctags, gdb }:
 
-stdenv.mkDerivation rec {
-  pname = "gede";
-  version = "2.14.1";
+mkDerivation rec {
+  name = "gede-${version}";
```

I just noticed it should be `pname` and `version` as it was previously.
```suggestion
  pname = "gede";
```

---

Yes, thanks.
I fixed `pname` and `url`.